dependencies/pom.xml

```diff
@@ -70,12 +70,12 @@
         <!-- This is the BOM version for Jackson Databind 2.13.4.2 -->
         <version.lib.jackson>2.13.4.20221013</version.lib.jackson>
         <version.lib.jakarta.activation-api>2.0.1</version.lib.jakarta.activation-api>
-        <version.lib.jakarta.annotation-api>2.0.0</version.lib.jakarta.annotation-api>
-        <version.lib.jakarta.cdi-api>3.0.0</version.lib.jakarta.cdi-api>
+        <version.lib.jakarta.annotation-api>2.1.0</version.lib.jakarta.annotation-api>
+        <version.lib.jakarta.cdi-api>4.0.1</version.lib.jakarta.cdi-api>
```

Comment:
This looks wrong. Helidon 3 is Jakarta EE 9, you are bumping specs to EE 10. As far as I know we should be sticking with Weld 4 / CDI 3  in Helidon 3.